packages/desktop/features/onboarding.features.ts

```diff
@@ -65,16 +65,16 @@ const onboardingFeaturesForIotaAlphanet: IOnboardingFeaturesForNetwork = {
         },
     },
     claimRewards: {
-        enabled: false,
-        hidden: true,
+        enabled: true,
```

Comment:
just one more thing :pray: 
I think that `claimRewards` doesn't need to be changed :see_no_evil: